wolfssl/wolfcrypt/logging.h

```diff
@@ -152,6 +153,28 @@ WOLFSSL_API void wolfSSL_SetLoggingPrefix(const char* prefix);
     #define WOLFSSL_TIME(n)  WC_DO_NOTHING
 #endif
 
+#if defined(XVSNPRINTF) && !defined(NO_WOLFSSL_MSG_EX) && \
+    (defined(DEBUG_WOLFSSL) || defined(WOLFSSL_DEBUG_CERTS))
+
+    #ifndef WOLFSSL_MSG_CERT_INDENT
+    #define WOLFSSL_MSG_CERT_INDENT "\t-  "
+    #endif
+
+    #ifdef __clang__
+    /* tell clang argument 1 is format */
+    __attribute__((__format__ (__printf__, 1, 0)))
+    #endif
+    WOLFSSL_API void WOLFSSL_MSG_CERT(const char* fmt, ...);
+#else
+    #undef WOLFSSL_DEBUG_CERTS
+    #ifdef WOLF_NO_VARIADIC_MACROS
+        /* More than 1 param must be wrapped with WOLFSSL_DEBUG_CERTS */
+        #define WOLFSSL_MSG_CERT(x)   WC_DO_NOTHING
```

Comment:
This should be `#define WOLFSSL_MSG_CERT()   WC_DO_NOTHING`. Or are you saying that ESP32 compiler threw an error with empty ()?

Reply:
Jim, you won't use `WOLF_NO_VARIADIC_MACROS` with the ESP32 compiler. Please use the suggestion I made... `#define WOLFSSL_MSG_CERT()   WC_DO_NOTHING`. It is to appease compilers that cannot handler `...` vardantic arguments. You don't need that option to pass on your compiler.